wiki/snapshot.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Snapshots
+layout: wiki
+---
+
+# Overview
+Snapshots are a snapshot of the chain at a particular blockheight.
+Gridcoin snapshots are created by trusted developers and they allow newer users to get into sync faster than syncing from 0.
```

Comment:
Snapshots being created by trusted developers is not necessarily true. Anyone can create a snapshot if they want to. The snapshots on `snapshot.gridcoin.us` are probably what you are referring to, but I think it may just be better to remove that part of the sentence

```suggestion
Snapshots can help newer users to get into sync faster than syncing from 0.
```